lib/drivers/medtronicDriver.js

```diff
@@ -908,7 +1039,7 @@ module.exports = function (config) {
 
       var sessionInfo = {
         deviceTags: ['insulin-pump'],
-        deviceManufacturers: ['Medtronic'],
+        deviceManufacturers: ['CareLink'], //TODO: change to Medtronic once viz is fixed
```

Comment:
Viz should be fixed, for device settings at least, unless there's other things that use the manufacturer that I have (clearly) forgotten. So perhaps remove this and LMK if something explodes? ;)